### ISSUE TITLE
Feature Request: Support typoscript Joins

### ISSUE THREAD

**Issue by Shupal** [markussom/sitemap_generator#20]
Hi markus,

thanks for the extension. It is really great and easy to use.

In my case i need a join between two tables for generated a sitemap for custom records. So is there any plans to support the typoscript select joins in the future?
Or a new AdditionalJoinService? ;-)

**Comment by christophlehmann** [comment]
I don't know your exact use case, but you can realize it with TypoScript, see https://docs.typo3.org/typo3cms/TyposcriptReference/ContentObjects/Content/Index.html and
https://docs.typo3.org/typo3cms/TyposcriptReference/Functions/Select/Index.html#select

**Comment by Shupal** [comment]
yes but i don't know if it is working as record configuration for the sitemap.
Cause we have here a custom configuration and not a core typoscript function.
